config/routes.rb

```diff
@@ -1,9 +1,13 @@
 Rails.application.routes.draw do
-  resources :entities
-  resources :groups
+  
+  devise_for :users, controllers: { sessions: 'users/sessions' }
+  resources :groups do
+    resources :entities
+  end
   resources :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root "splashes#index"
```

Comment:
- [ ]  when I sign up successfully it should take me to the home page categories  right now it is a bit complicated you have to click login some users won't know that so please try to fix that